spec/integration/client_spec.rb

```diff
@@ -0,0 +1,66 @@
+require "integration/helper"
+require "portus/registry_client"
+
+def push_test_images
+  name = ldap? ? "johnldap" : "john"
+  email = "john@example.com"
+  password = "12341234"
+  create_user(name, email, password, true)
+  expect { login(name, password, email) }.not_to raise_error
+
+  # Pulling images that we should already have (so we go faster :P). Then
+  # re-tag it so they can be pushed. We tag both a pusheable image and
+  # another one that cannot be pushed (namespace does not exist).
```

Comment:
Outdated comment is outdated. I used to do something else here. I'll also remove this "another" image because it used to be used for something else and now we discard it completely (while slowing down tests...).